content/en/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -100,52 +100,50 @@ Here is a configuration file you can use to create a Pod:
 
 1. Create the Pod:
 
-    ```shell
-    kubectl apply -f https://k8s.io/examples/pods/inject/secret-pod.yaml
-    ```
+   ```shell
+   kubectl apply -f https://k8s.io/examples/pods/inject/secret-pod.yaml
+   ```
 
 1. Verify that your Pod is running:
 
-    ```shell
-    kubectl get pod secret-test-pod
-    ```
+   ```shell
+   kubectl get pod secret-test-pod
+   ```
 
-    Output:
-    ```shell
-    NAME              READY     STATUS    RESTARTS   AGE
-    secret-test-pod   1/1       Running   0          42m
-    ```
+   Output:
+   ```
+   NAME              READY     STATUS    RESTARTS   AGE
+   secret-test-pod   1/1       Running   0          42m
+   ```
 
 1. Get a shell into the Container that is running in your Pod:
-    ```shell
-    kubectl exec -it secret-test-pod -- /bin/bash
-    ```
+   ```shell
+   kubectl exec -i -t secret-test-pod -- /bin/bash
+   ```
 
 1. The secret data is exposed to the Container through a Volume mounted under
-`/etc/secret-volume`. In your shell, go to the directory where the secret data
-is exposed:
-    ```shell
-    root@secret-test-pod:/# cd /etc/secret-volume
-    ```
+`/etc/secret-volume`.
 
-1. In your shell, list the files in the `/etc/secret-volume` directory:
-    ```shell
-    root@secret-test-pod:/etc/secret-volume# ls
-    ```
-    The output shows two files, one for each piece of secret data:
-    ```shell
-    password username
-    ```
+   In your shell, list the files in the `/etc/secret-volume` directory:
+   ```shell
+   # Run this in the shell inside the container
+   ls /etc/secret-volume
+   ```
+   The output shows two files, one for each piece of secret data:
+   ```
+   password username
+   ```
 
 1. In your shell, display the contents of the `username` and `password` files:
-    ```shell
-    root@secret-test-pod:/etc/secret-volume# cat username; echo; cat password; echo
-    ```
-    The output is your username and password:
-    ```shell
-    my-app
-    39528$vdg7Jb
-    ```
+   ```shell
+   # Run this in the shell inside the container
+   cat /etc/secret-volume/username; echo; cat /etc/secret-volume/password; echo
```

Comment:
Hello @sftim 
Changes look good.
nit: If you list the cat/echo commands on separate lines, will both echo commands render in the highlight color?
nit: At some point, I would like to see the `References` section appear before the `What's next` section.